static/src/javascripts/projects/commercial/modules/dfp/prepare-googletag.ts

```diff
@@ -59,18 +59,24 @@ const setDfpListeners = (): void => {
 	pubads.addEventListener('slotVisibilityChanged', onSlotVisibilityChanged);
 
 	if (storage.session.isAvailable()) {
-		const pageViews =
-			(storage.session.get('gu.commercial.pageViews') as number) || 0;
-		storage.session.set('gu.commercial.pageViews', pageViews + 1);
+		const pageViews: unknown = storage.session.get(
+			'gu.commercial.pageViews',
+		);
+
+		// Increment only if positive number and not NaN, otherwise set to 0
+		typeof pageViews === 'number' && pageViews >= 0
+			? storage.session.set('gu.commercial.pageViews', pageViews + 1)
+			: storage.session.set('gu.commercial.pageViews', 0);
 	}
 };
 
 const setPageTargeting = (): void => {
 	const pubads = window.googletag.pubads();
 	// because commercialFeatures may export itself as {} in the event of an exception during construction
-	const targeting = getPageTargeting() as Record<string, string | string[]>;
-	Object.keys(targeting).forEach((key) => {
-		pubads.setTargeting(key, targeting[key]);
+	const targeting = getPageTargeting();
```

Comment:
V.minor - can we inline `pubads` and `targeting`? I don't think we wouldn't lose much readability?